README.md

```diff
@@ -4,7 +4,10 @@
 `0.0.1`
 
 ## Purpose
-This repository serves as a documentation around WordPress hosting best practices and considerations. It is intended for folks responsible for the infrastructure of a WordPress site or sites.
+This repository serves as a documentation around WordPress hosting best practices and considerations. It is intended for folks responsible for the infrastructure of a WordPress site or sites. For each topic, the documentation will provide an overview someone might need to know, as it relates to WordPress.
+
+## Audience
+This documentation is for technical folks hosting WordPress for business critical needs (client sites, company sites, high traffic personal site, etc.). They may be newer to WordPress and/or hosting, such as starting a hosting business, but aren't totally new to technology in general and would be able to understand concepts easily but need a resource where those best practices are laid out. It is not oriented at end users who aren't comfortable with deeper technical topics.
```

Comment:
I don't think we need to restrict ourselves to "business critical needs". Rather, we can delineate topics (and suggestions) based on the type of site it is. For instance, "object caching" is relevant to everyone, because it's a key aspect to how WordPress works. Most standard sites are fine without persistent caching, but it's highly relevant for sites with complex data needs.